app/controllers/application_controller/ci_processing.rb

```diff
@@ -1619,6 +1619,42 @@ def scanhosts
     host_button_operation('scan', 'Analysis')
   end
 
+  # Scan all selected or single displayed image(s)
+  def scanimages
+    assert_privileges("image_scan")
+    images = Array.new
+    if @lastaction == "show_list"
+      images = find_checked_items
+    else
+      if params[:id].nil? || ContainerImage.find_by_id(params[:id]).nil?
+        add_flash(_("%s no longer exists") % ui_lookup(:table=>"container_image"), :error)
+      else
+        images.push(params[:id])
+      end
+    end
+    _log.info("Container images scan list is #{images}")
+    if images.empty?
+      add_flash(_("No %{model} were selected for %{task}") % {:model=>ui_lookup(:tables=>"container_image"), :task=>"Analysis"}, :error)
+    else
+      ContainerImage.where(:id => images).order("lower(name)").each do |image|
+        image_name = image.name
+        begin
+          image.scan
+        rescue StandardError => bang
+          add_flash(_("%{model} \"%{name}\": Error during 'Analysis': ") % {:model=>ui_lookup(:model=>"ContainerImage"), :name=>image_name} << bang.message, :error) # Push msg and error flag
```

Comment:
This is the convention which is used (see lines 1492, 1464 and 1582 for example)